README.md

```diff
@@ -22,7 +22,8 @@ Now you're ready to connect to your SSE endpoint and start streaming updates!
 :godmode:
 
 ```js
-const eventSource = new RNEventSource('https://my-sse.com/stream');
+const options = { headers: { Authorization: 'Baerer ...' } };
+const eventSource = new RNEventSource('https://my-sse.com/stream', options);
```

Comment:
Since the `options` argument is optional, the README should make a note of that to help guide new users of the library. I'd prefer to omit the options from the first code snippet, and add another block of text explaining that headers can be passed as a second argument to the constructor of `EventSource`